extmod/network_ninaw10.c

```diff
@@ -849,12 +849,15 @@ STATIC const mp_rom_map_elem_t nina_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_status),              MP_ROM_PTR(&network_ninaw10_status_obj) },
     { MP_ROM_QSTR(MP_QSTR_ioctl),               MP_ROM_PTR(&network_ninaw10_ioctl_obj) },
 
+    // Class constants.
+    { MP_ROM_QSTR(MP_QSTR_STA_IF),              MP_ROM_INT(MOD_NETWORK_STA_IF) },
+    { MP_ROM_QSTR(MP_QSTR_AP_IF),               MP_ROM_INT(MOD_NETWORK_AP_IF) },
     // Network is not secured.
-    { MP_ROM_QSTR(MP_QSTR_OPEN),                MP_ROM_INT(NINA_SEC_OPEN) },
+    { MP_ROM_QSTR(MP_QSTR_AUTH_OPEN),           MP_ROM_INT(NINA_SEC_OPEN) },
     // Security type WEP (40 or 104).
-    { MP_ROM_QSTR(MP_QSTR_WEP),                 MP_ROM_INT(NINA_SEC_WEP) },
+    { MP_ROM_QSTR(MP_QSTR_AUTH_WEP),            MP_ROM_INT(NINA_SEC_WEP) },
     // Network secured with WPA/WPA2 personal(PSK).
-    { MP_ROM_QSTR(MP_QSTR_WPA_PSK),             MP_ROM_INT(NINA_SEC_WPA_PSK) },
+    { MP_ROM_QSTR(MP_QSTR_AUTH_WPA_PSK),        MP_ROM_INT(NINA_SEC_WPA_PSK) },
```

Comment:
Is this mixed mode?  `SEC_WPA_WPA2`.

Reply:
When starting the AP mode, it looks like only WEP is supported (in our driver, `nina_start_ap` only supports the open and WEP constants).

But the command we send to the Nina doesn't actually specify the mode. Our driver and the Arduino driver use the same command (0x19, which they call `SET_AP_PASSPHRASE_CMD` and document as WPA, and we call `NINA_CMD_START_AP_WEP`).

Reply:
> Our driver and the Arduino driver use the same command

No we use different commands: https://github.com/micropython/micropython/blob/master/drivers/ninaw10/nina_wifi_drv.c#L402
It's seems that WPA is also supported in AP mode, in the firmware, but there's no command to actually connect use it. Maybe I will push a firmware update later to fix it.

> Is this mixed mode? `SEC_WPA_WPA2`.

@dpgeorge The security mode is not set in station mode, it seems to connect to all modes, except for deprecated/unsecure ones like WEP.